src/counts.py

```diff
@@ -9,6 +9,16 @@
 # Your code goes here.
 
 
+
+for i in range(len(x)):
+    char=x[i]
```

Comment:
A better approach is

```python
for char in x:
   ...
```

To iterate through the characters in a string you don't need to go via the indices. Iterating through the string directly is also generally faster.